Application/Source/Models/Content.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+import RxCocoa
+
+protocol Content {
```

Comment:
Do we need this protocol at this time?

It looks like we only need the `Food` enum without a protocol abstraction for now. Perhaps we should remove the protocol to reduce complexity in this PR?